VSTS/SonarQube/.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
```

Comment:
Comments are not valid in JSON. You can add an additional property like this for comments: `"comment": "comment text here"`.